packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -40,6 +40,8 @@ BuildArch: noarch
 
 BuildRequires: sed
 
+Obsoletes: foreman-release-scl < 7-3
```

Comment:
Just wondering if everyone who installed `foreman-release-scl` also installed `foreman-release`. Perhaps it'd be safer to obsolete it in `tfm`?

Reply:
Yea, suppose so. Dunno what we rae gonna do when we get to pure EL8 and we don't have a tfm package as a catch all for cleaning up user systems.

Reply:
Then we'll have less things to obsolete since unused gems are no longer in our own SCL so they could be used by other tools. We can introduce an optional foreman-obsoletes similar to how Fedora removes obsolete packages.